lib/modules/submitHelper.js

```diff
@@ -51,41 +58,53 @@ const $repostWarning = _.once(() => $(string.html`
 let urlField, srField;
 
 module.go = () => {
-	if (isPageType('submit') && module.options.warnAlreadySubmitted.value) {
-		const urlFieldDiv = document.querySelector('#url-field');
-		if (urlFieldDiv) {
-			$(urlFieldDiv).parent().after($repostWarning());
-			urlField = ((urlFieldDiv.querySelector('#url'): any): HTMLInputElement);
-			srField = ((document.querySelector('#sr-autocomplete'): any): HTMLInputElement);
-			$([srField, urlField]).on('input keydown', _.debounce(updateRepostWarning, 300));
-			// No event is fired when reddit's js changes the subreddit field, so update whenever the user clicks
-			$('#suggested-reddits a, #sr-drop-down').on('click', updateRepostWarning);
-			// We would allow reddit to show/hide the message for link/text posts with #link-desc
-			// but some subreddits hide this box, so we'll do it manually.
-			const linkButton = document.querySelector('a.link-button');
-			const textButton = document.querySelector('a.text-button');
-			if (linkButton && textButton) {
-				linkButton.addEventListener('click', () => { updateRepostWarning(); });
-				textButton.addEventListener('click', () => { $repostWarning().hide(); });
-			}
-		}
+	implementOptions();
+	// Register submit CLI if not on the submit page
+	if (!isPageType('submit')) {
+		registerCommandLine();
 	}
+};
 
-	if (isPageType('submit') && module.options.uncheckSendRepliesToInbox.value) {
-		const sendReplies: ?HTMLInputElement = (document.querySelector('#sendreplies'): any);
-		if (sendReplies) {
-			sendReplies.checked = false;
+async function implementOptions() {
+	if (isPageType('submit')) {
```

Comment:
To avoid unnecessary many levels of nesting, return early (`if (!isPageType('submit')) return;`).

Reply:
Yeah, good point. Thanks!